phone/src/Phone.tsx

```diff
@@ -104,9 +104,10 @@ function Phone() {
               id="phone"
               className="PhoneScreen"
               style={{
-                backgroundImage: settings.customWallpaper
-                  ? `url(${settings.customWallpaper})`
-                  : `url(media/backgrounds/${settings.wallpaper.value})`,
+                backgroundImage:
+                  settings.wallpaper.label === 'Custom Wallpaper'
```

Comment:
what happens in other languages?

Reply:
Well...that was the issue that I was preparing you for, since the 'Custom Wallpaper' is in the default.json file, I need to figure out a way to fix it, or the users would have to change this manually. I am not quite sure how to do this, or atleast not the best way. So I want some input on this.